### PR TITLE
Move Foreman::Renderer from lib/ to app/services

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -650,6 +650,7 @@ Style/DoubleNegation:
     - 'app/models/template.rb'
     - 'app/registries/foreman/settings/auth.rb'
     - 'app/services/foreman/render_templates_from_folder.rb'
+    - 'app/services/foreman/renderer/scope/variables/base.rb'
```
Comment (ezr-ondrej) [inline .rubocop_todo.yml:653]: few lines below you've left the exception, for what is now non-existent file, could you remove that (`lib/foreman/renderer...`)

**Review by ezr-ondrej — DISMISSED** [review]
Make sense IMHO, just one nit :)

**Comment by stejskalleos** [comment]
Correct RM: https://projects.theforeman.org/issues/34139

**Review by ezr-ondrej — APPROVED** [review]
I hope this is not a breaking change and tests are green :+1:

**Pull request merged**